requirements-webapp.txt

```diff
@@ -16,6 +16,7 @@ marshmallow
 numpy
 oauthlib
 openapi-spec-validator
+prometheus-client
```

Comment:
This was an implicit dependency of `prometheus-flask-exporter`. Better to make it implicit.

Not so important. At some point I think we will have all dependencies explicitly locked down (pinned) anyway.